server.js

```diff
@@ -3293,6 +3293,7 @@ cache(function(data, match, sendBadge, request) {
       // we'll render the 'invalid' badge below, which is the correct thing
       // to do.
       var version = versionLines[0].replace(/\s+/, '').split(/:/)[1];
+      version = version.trim();
```

Comment:
What do you think if we just use something like:
```javascript
var version = versionLines[0].split(/:/)[1].trim();
```

Reply:
that's neater - yes :)